crates/starknet-server/src/api/json_rpc/error.rs

```diff
@@ -38,8 +38,14 @@ pub enum ApiError {
     InvalidContractClass,
     #[error("Only latest/pending block is supported")]
     OnlyLatestBlock,
-    #[error("Unsupported version")]
-    UnsupportedVersion,
+    #[error("{msg}")]
+    UnsupportedAction { msg: String },
+    #[error(transparent)]
+    TransactionError(#[from] starknet_in_rust::transaction::error::TransactionError),
+    #[error(transparent)]
+    SyscallHandlerError(
```

Comment:
the same here

Reply:
Resolved in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/99/commits/c3f070b62ff1982a46b8f4ac42c92e2cbb1e6cf3